apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,61 @@
+h1, body, html, main {
+  margin: 0;
+}
+/*Import the font and set the default size*/
+html {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 18px
+}
+.header {
+  margin: 0 auto;
+  color: white; 
+  text-align: center;
+  vertical-align: middle;
+  text-size: 100px;
+}
+.cover-image {
+  background-image: url("/Users/tanjameinert/Desktop/ironhack/clone-test/lab-css-recipes-clone/apple-pie-starter-code/images/apple-pie.jpg");
+  background-position: 0 0;
+  background-repeat: no-repeat;
+  background-size: cover;
+  line-height: 500px;
+}
+
+#intro {
+  text-align: center;
+  color: dimgray;
+  font-size: 20px;
+  margin: 20px 0 20px 0;
+}
+
+hr{
+  /*margin-bottom: 30px;
```

Comment:
Nooo. If you need to create sapces between content in you page use the ```class``` or ```id``` of that one. Not add ```hr``` and style this one.